src/pkg/projects/Microsoft.NETCore.App/netcoreapp1.0.props

```diff
@@ -13,10 +13,13 @@
       <Version>4.0.1</Version>
     </NETCoreApp10Dependency>
     <NETCoreApp10Dependency Include="Microsoft.NETCore.DotNetHostPolicy">
-      <Version>1.0.1</Version>
+      <Version>1.0.5</Version>
     </NETCoreApp10Dependency>
     <NETCoreApp10Dependency Include="Microsoft.NETCore.Runtime.CoreCLR">
-      <Version>1.0.4</Version>
+      <Version>1.0.7</Version>
+    </NETCoreApp10Dependency>
+    <NETCoreApp10Dependency Include="Microsoft.NETCore.Targets">
```

Comment:
Also, 1.0.5 has a dependency on runtime.native.System.Security.Cryptography (>= 4.0.1) now.

---
In reply to: [118324972](https://github.com/dotnet/core-setup/pull/2515#discussion_r118324972) [](ancestors = 118324972)